### PR TITLE
Event Attendance additions

### PR DESCRIPTION
**Pull request by sleidig**

### CODE REVIEW

File: src/app/core/entity-components/entity-details/form/form.component.ts [hunk 1]
```diff
@@ -98,6 +98,7 @@ export class FormComponent implements OnInitDynamicComponent {
     const formConfig = {};
     this.config.cols.forEach((c: FormFieldConfig[]) =>
       c.forEach((r) => {
+        console.log("field", r);
```
Comment (TheSlimvReal) [inline src/app/core/entity-components/entity-details/form/form.component.ts:101]: remove?

File: src/app/core/form-dialog/form-dialog.service.ts [hunk 2]
```diff
@@ -24,14 +26,27 @@ export class FormDialogService {
...
     const dialogRef = this.dialog.open(entityDetailsComponent, {
       width: "80%",
     });
+
     dialogRef.componentInstance.entity = entity;
+    if (
+      typeof (dialogRef.componentInstance as OnInitDynamicComponent)
+        .onInitFromDynamicConfig === "function"
+    ) {
+      (dialogRef.componentInstance as OnInitDynamicComponent).onInitFromDynamicConfig(
```
Comment (TheSlimvReal) [inline src/app/core/form-dialog/form-dialog.service.ts:46]: Maybe a test that ensures that the function is actually called when an appropriate component is passed

**Review by TheSlimvReal — CHANGES_REQUESTED** [review]
Very nice changes, just some minor notes

File: src/app/core/form-dialog/form-dialog.service.ts (hunk 2 above)
Comment (sleidig) [reply src/app/core/form-dialog/form-dialog.service.ts:46]: added in the core PR https://github.com/Aam-Digital/ndb-core/pull/604

**Pull request merged**